roles/openshift_cloud_provider/tasks/gce.yml

```diff
@@ -27,6 +27,7 @@
     value: "{{ item.value }}"
   with_items:
   - { key: 'project-id', value: '{{ openshift_gcp_project }}' }
+  - { key: 'network-project-id', value: '{{ openshift_gcp_network_project }}' }
```

Comment:
This should have `openshift_gcp_network_project | default(openshift_gcp_project)`